src/components/DataFetchingInReact/DataFetchingInReact.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import "./DataFetchingInReact.css";
+
+const DataFetchingInReact = () => {
+  const [users, setUsers] = useState(null);
```

Comment:
I think the default value should be an empty array because that is the data type that we are expecting from the API.

Reply:
Thanks, @lycoangelo. This is now updated.